### ISSUE TITLE
It looks like metadata is quoted, is that the intended behavior?

### ISSUE THREAD

**Issue by sfabriece** [creditkarma/good-influx#18]
Quoted metadata is breaking my metrics processor. Is it possible to unquote all the metadata in line with how the value of host is processed.

currently:

```
ops,host=myhost,pid=12992,serviceName="api",environment="development" os.cpu1m=0,os.cpu5m=0,os.cpu15m=0
```

would prefer

```
ops,host=myhost,pid=12992,serviceName=api,environment=development os.cpu1m=0,os.cpu5m=0,os.cpu15m=0
```

I'm happy to PR this if this is reasonable.

**Comment by AlecHFerguson** [comment]
@sfabriece You're right, the docs recommend no quotes in tag values:
https://docs.influxdata.com/influxdb/v1.2/write_protocols/line_protocol_tutorial/#quoting

```
Do not double or single quote measurement names, tag keys, tag values, and field keys. It is valid Line Protocol but InfluxDB assumes that the quotes are part of the name.
```

It sounds like if you assume the quotes are part of the string then you should be able to query it?  If you query like this?

```
> SELECT * FROM "ops" WHERE "serviceName" = "\"api\""
```

As a workaround, can you try this in your metrics processor?

PR would be welcome.  Probably just don't call `formatString` on the tag values?
https://github.com/fhemberger/good-influx/blob/master/lib/formatters.js#L37

Consider whether we should handle special characters like spaces, equals signs, commas, etc., by adding a backslash before them?  Just a thought, not sure what the right answer is.  Also, the way we've recently split out tests should make testing easier.

**Comment by fhemberger** [comment]
Yes, a PR would be welcome. 

Do we need an additional check instead if someone tries to add a value with a space in it?

**Comment by sfabriece** [comment]
I'll look at the docs and create an initial PR, then we can discuss any changes.

**Comment by AlecHFerguson** [comment]
Sounds good @sfabriece. I'd be happy to help review. I'll be off in the wilderness for a few days but will definitely take a look when I get back.

**Comment by AlecHFerguson** [comment]
@sfabriece Have you had a chance to look into this? I might be able to work on a fix if not.

**Comment by sfabriece** [comment]
I've been quite busy lately. Would appreciate it if you could do it @AlecHFerguson .

**Comment by AlecHFerguson** [comment]
@sfabriece Just opened a PR. Mind taking a look? Also, if you could test this on your local instance that would be swell.

**Comment by fhemberger** [comment]
Published `good-influx@2.3.1`
